coral-component-list/src/scripts/SelectList.js

```diff
@@ -277,14 +305,14 @@ class SelectList extends BaseComponent(HTMLElement) {
   /** @private */
   _focusPreviousItem(event) {
     event.preventDefault();
-    
+    event.stopPropagation();
```

Comment:
Why do we need to stop the event?
If this is required then shouldn't we stop it for - _onHomeKey(_focusFirstItem), _onEndKey(_focusLastItem)